spec/lex.dd

```diff
@@ -13,6 +13,24 @@ phase of translation.)
 
 $(H2 $(LNAME2 source_text, Source Text))
 
+$(GRAMMAR
+$(GNAME SourceFile):
+    $(GLINK ByteOrderMark) $(GLINK Module)$(OPT)
```

Comment:
`$(GLINK Module)` yields a dead link.

Reply:
Does a leading byte-order mark really preclude a following shebang?
Otherwise this rule could be simplified to
```
ByteOrderMark (opt) Shebang (opt) Module (opt)
```

Reply:
Yes, both the byte order mark and shebang want to be the very first bytes in the physical file. So, you can't have both.